src/elements/content-sidebar/activity-feed/annotations/__tests__/AnnotationActivityMenu.test.js

```diff
@@ -21,6 +21,12 @@ describe('elements/content-sidebar/ActivityFeed/annotations/AnnotationActivityMe
         },
     );
 
+    test('should render the edit annotation activity menu item if canEdit is true', () => {
+        const wrapper = getWrapper({ canEdit: true });
+
+        expect(wrapper.find('[data-testid="edit-annotation-activity"]').length).toEqual(1);
```

Comment:
You can also use `wrapper.exists` if the exact count isn't dynamic.

Reply:
Good call, will update.